packages/shorebird_cli/lib/src/commands/release_command.dart

```diff
@@ -250,6 +250,22 @@ ${summary.join('\n')}
 
     createArtifactProgress.complete();
 
+    final uploadAppBundleProgress = logger.progress('Uploading app bundle');
```

Comment:
rather than splitting this out since it technically is just "another artifact", wdyt of moving this as part of the `createArtifactProgress` step instead of defining a separate progress step? It could still run last but I don't think there's much value is letting the user know we're both uploading artifacts and and app bundle (since they are all artifacts as far as the end user is concerned imo). Let me know what you think.

Reply:
Makes sense, done.